src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyUser.php

```diff
@@ -53,6 +53,20 @@ class CompanyUser implements CompanyUserInterface
      */
     protected $companyUserCompanyAssignerRepository;
 
+    /**
+     * @var \Spryker\Zed\CompanyBusinessUnit\Business\CompanyBusinessUnitFacadeInterface
+     */
+    protected $companyBusinessUnitFacade;
+
+    /**
+     * @param \FondOfSpryker\Zed\CompanyUserCompanyAssigner\CompanyUserCompanyAssignerConfig $companyUserCompanyAssignerConfig
+     * @param \FondOfSpryker\Zed\CompanyUserCompanyAssigner\Persistence\CompanyUserCompanyAssignerRepository $companyUserCompanyAssignerRepository
+     * @param \Spryker\Zed\CompanyUser\Business\CompanyUserFacadeInterface $companyUserFacade
+     * @param \FondOfSpryker\Zed\Company\Business\CompanyFacadeInterface $companyFacade
+     * @param \FondOfSpryker\Zed\CompanyType\Business\CompanyTypeFacadeInterface $companyTypeFacade
+     * @param \Spryker\Zed\CompanyRole\Business\CompanyRoleFacadeInterface $companyRoleFacade
+     * @param \Spryker\Zed\CompanyBusinessUnit\Business\CompanyBusinessUnitFacadeInterface $companyBusinessUnitFacade
+     */
     public function __construct(
         CompanyUserCompanyAssignerConfig $companyUserCompanyAssignerConfig,
         CompanyUserCompanyAssignerRepository $companyUserCompanyAssignerRepository,
```

Comment:
@SpiderDan98 use the interface of repository.

Reply:
@daniel-rose I changed it and updated the tests